style.css

```diff
@@ -0,0 +1,628 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Poppins&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+.header-container {
+  background-color: #272a31;
+  padding: 5px 100px 5px 100px;
+  box-shadow: 5px 10px 18px #272727;
+}
+
+.hamburger {
+  display: none;
+  cursor: pointer;
+}
+
+.bar {
+  display: block;
+  width: 30px;
```

Comment:
- [OPTIONAL] Kindly make sure your project is responsive on desktop as the previous forgot to mention, I see that your design is responsive on <=768px but not beyond, Kindly fix to make your project accessible to many devices.

![Screenshot from 2023-02-17 23-37-52](https://user-images.githubusercontent.com/97834160/219799358-4f775088-0431-4770-a577-c89429615cf9.png)